packages/databricks-sdk-js/src/AccountClient.ts

```diff
@@ -282,12 +282,6 @@ export class AccountClient {
         }
 
         this.config = config as Config;
-
-        this.config.ensureResolved();
-        if (!this.config.accountId || !this.config.isAccountClient()) {
-            throw new Error("invalid Databricks Account configuration");
-        }
```

Comment:
Why did you move it out? I don't understand the use of moving it out.

Reply:
`ensureResolved` is async but since this is the constructor we can't await for this. 
This means that the check below never actually worked. I moved the check into the actual calls.

Reply:
Makes sense. Thanks!